tools/automatic_query_fixer/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'com.google.cloud.bigquery.utils'
+version '1.0-SNAPSHOT'
+
+
+
+application {
+    mainClassName = "com.google.cloud.bigquery.utils.auto_query_fixer.Application"
```

Comment:
i think the style should be "autoQueryFixer"?

Reply:
actually thinking what might be a better package name...

Reply:
decide to use queryfixer